app/src/main/java/com/android/feedme/model/viewmodel/GalleryViewModel.kt

```diff
@@ -21,15 +21,15 @@ import kotlinx.coroutines.flow.MutableStateFlow
 
 class GalleryViewModel : ViewModel() {
 
-  val _bitmaps = MutableStateFlow<List<Bitmap>>(emptyList())
```

Comment:
Should this val be private ?

Reply:
Nope, as mentioned in the other PR it is used in another class